### ISSUE TITLE
Windows pathname transcription from M2C to M2

### ISSUE THREAD

**Issue by BrianKnoblauch** [m2sf/m2pp#3]
Syntax diagrams for the Windows pathnames: 
https://github.com/m2sf/m2sharp/wiki/Pathname-Diagrams#windows-pathname-diagrams

M2C repository: https://github.com/trijezdci/m2c

M2 version will also need to go into the m2bsk repository

**Comment by BrianKnoblauch** [comment]
@trijezdci In the m2bsk repo I see that pathnames are in the src\lib\Pathnames directory.  That does not exist in m2pp.  Should the contents of that be copied over from m2bsk or is the intent on having a different structure?  While I had planned on doing this in m2pp first, I'm now wondering if I should do in m2bsk first so that we can bulk move everything over to m2pp in one shot?

**Comment by trijezdci** [comment]
>
> In the m2bsk repo I see that pathnames are in the src\lib\Pathnames
> directory. That does not exist in m2pp. Should the contents of that be
> copied over from m2bsk or is the intent on having a different structure?
>
Correct. Remember I had restructured the M2BSK source tree a month or so
ago. Eventually we should restructure the M2PP source tree in the same way,
but I left that alone for the time being while I focus on M2BSK.


> While I had planned on doing this in m2pp first, I'm now wondering if I
> should do in m2bsk first so that we can bulk move everything over to m2pp
> in one shot?
>
Good thinking. Let's do it that way.
